woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -965,3 +963,32 @@ def test_nullable_attribute(sample_df_pandas):
         expected = EXPECTED_COLUMN_NULLABILITIES[key]
 
         assert actual is expected
+
+
+def test_validate(sample_df):
+    series = sample_df["email"]
+    series = init_series(series, logical_type="EmailAddress")
+    assert series.ww.validate() is None
+
+    invalid_row = pd.Series({4: "bad_email"}, name="email", dtype="string")
+
+    if _is_koalas_series(series):
+        invalid_row = ks.from_pandas(invalid_row)
+
+    series = series.append(invalid_row)
+    series = init_series(series, logical_type="EmailAddress")
+    match = "Series email contains invalid email addresses."
+
+    with pytest.raises(ValueError, match=match):
+        series.ww.validate()
+
+    actual = series.ww.validate(return_invalid_values=True)
+
+    if _is_dask_series(actual):
+        actual = actual.compute()
+
+    if _is_koalas_series(actual):
+        actual = actual.to_frame().to_pandas()["email"]
```

Comment:
Use the `to_pandas` utility function here?

Reply:
updated